### PR TITLE
Feature/randomised daily biometric notifications

### PR DESCRIPTION
**Pull request by Fralez**
In this PR there is added:

- [ X ] Randomification of daily notifications
- [ X ] Automatic rearranging of daily notification randomness day by day
- [ X ] Instant notification need of notification id specification
- [ X ] Service models working with local server from Android emulator
- [ X ] Removal of unused imports and useless comments

### CODE REVIEW

File: lib/services/LocalNotificationService.dart [hunk 1]
```diff
@@ -70,8 +77,9 @@ Future<void> onSelectNotification(String payload) {
   if (payload != null) {
     print('notification payload: ' + payload);
   }
-  if (payload == 'doBiometrics') {
-    locator<NavigationService>().navigateTo('alertpagebiometrics');
+  if (payload.split("-")[0] == 'doBiometrics') {
```
Comment (cherta) [inline lib/services/LocalNotificationService.dart:80]: `startsWith` again here

File: lib/services/models/AuthenticationProof.dart [hunk 2]
```diff
@@ -1,12 +1,11 @@
 import 'dart:async';
 import 'dart:convert';
-import 'package:flutter_background_geolocation/flutter_background_geolocation.dart';
 import 'package:http/http.dart' as http;
 import 'package:shared_preferences/shared_preferences.dart';
 
 import 'package:bien_aca_quarantine/services/models/User.dart';
 
-final serverUrl = 'http://localhost:3000';
+final serverUrl = 'http://10.0.2.2:3000';
```
Comment (cherta) [inline lib/services/models/AuthenticationProof.dart:8]: Change back this to server URL

File: lib/services/models/User.dart [hunk 3]
```diff
@@ -4,10 +4,10 @@ import 'package:http/http.dart' as http;
 import 'package:shared_preferences/shared_preferences.dart';
 
 import 'package:bien_aca_quarantine/services/GeofencingService.dart';
-import 'package:bien_aca_quarantine/services/LocalNotificationService.dart';
 
 enum Gender { MALE, FEMALE }
-final serverUrl = 'https://bian-aca-prod.herokuapp.com';
+final serverUrl = 'http://10.0.2.2:3000';
```
Comment (cherta) [inline lib/services/models/User.dart:9]: And this

File: lib/components/pages/home_page.dart [hunk 4]
```diff
@@ -53,8 +66,9 @@ class _HomePageState extends State<HomePage> {
     });
     initializeLocalNotifications(_onDidReceiveLocalNotification,
         onSelectNotification: (String payload) {
-      if (payload == 'doBiometrics') {
-        locator<NavigationService>().navigateTo('alertpagebiometrics');
+      if (payload.split("-")[0] == 'doBiometrics') {
```
[the remark this answers is not in the document]
Comment (Fralez) [reply lib/components/pages/home_page.dart:69]: Didn't know the method, I'll update it ASAP!

File: lib/services/models/AuthenticationProof.dart (hunk 2 above)
Comment (Fralez) [reply lib/services/models/AuthenticationProof.dart:8]: This actually was on localhost, not on server URL. I'll change all local URL back to heroku server

File: lib/services/models/User.dart (hunk 3 above)
Comment (Fralez) [reply lib/services/models/User.dart:9]: IDEM the first request

**Pull request merged**